client/src/components/Footer/footer.css

```diff
@@ -0,0 +1,50 @@
+.section2 {
```

Comment:
Rename your classes to something more readable. And generalize properties like display and alignment to separate classes, since you use those a lot.